### ISSUE TITLE
project-page

### ISSUE THREAD

**Issue by isabel-amon** [hpi-swt2/vm-portal#166]
[//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)

**User Story**

[//]: # (Be descriptive about your user story. Who requests a feature and what's his intent?)

As a user, I want to be able to see details of a project. When I click on a project name on the all-projects page (#164) or under "my projects" on my profile page (#168) and then get to the detail page. On it I want to read the name of the project, the responsible people and team members, as well as the description of the project. If I am a responsible person I want to have a button allowing me to edit the project. I also want a button that allows me to go back to the all-projects page.

**Acceptance Criteria:**

- [ ] On a page I find the name of the project, the people responsible, the teammembers and the purpose of the project
- [ ] If I am the responsible person I have a button that lets me edit the project
- [ ] A button exists that leads back to the all-projects page

[//]: # (Please don't forget to add the priority and complexity label to this issue.)

**Comment by skydivin4ng3l** [comment]
blocked by #163

**Comment by janwestphal306** [comment]
@isabel-amon there already is a link in the _header_ that leads to the projects overview. Should there still be an extra link on this page?

**Comment by isabel-amon** [comment]
I would still like to have another option to go back

**Comment by lassesteffen** [comment]
@isabel-amon Who are 'the teammembers' and what is their difference to responsible_users?

**Comment by lassesteffen** [comment]
@isabel-amon I am just going to add an button to edit the project. If you want to have a full edit page implemented for the project please add another issue.

**Comment by isabel-amon** [comment]
@lassesteffen the teammembers were removed on wish of the customer, I forgot to take them out of this issue, sorry.
An edit-button is fine.
